### PR TITLE
ARMADA-712 - Enhance Jobbergate's appearance on PyPI

### PR DESCRIPTION
**Pull request by fschuch**
#### What
Some work needs to be done around the README files at Jobbergate. Especially after their release to PyPI (where the readme files are shown).

In addition, more metadata can be added to the `pyproject.toml` files to improve the user experience.

#### Why
Enhance Jobbergate's appearance on PyPI

`Task`: https://app.clickup.com/t/18022949/ARMADA-712

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

File: jobbergate-api/README.rst [hunk 1]
```diff
@@ -1,3 +1,47 @@
+.. raw:: html
+
+   <p align="center">
+     <img
+       src="https://img.shields.io/github/workflow/status/omnivector-solutions/jobbergate/Test/main?label=main-build&logo=github&style=plastic"
+       alt="main build"
+     />
+     <img
+       src="https://img.shields.io/github/issues/omnivector-solutions/jobbergate?label=issues&logo=github&style=plastic"
```
Comment (dusktreader) [inline jobbergate-api/README.rst:9]: I think the issues and prs badges should be left at the top level, not on the subproject.

File: jobbergate-cli/README.rst [hunk 2]
```diff
@@ -1,3 +1,47 @@
+.. raw:: html
+
+   <p align="center">
+     <img
+       src="https://img.shields.io/github/workflow/status/omnivector-solutions/jobbergate/Test/main?label=main-build&logo=github&style=plastic"
+       alt="main build"
+     />
+     <img
+       src="https://img.shields.io/github/issues/omnivector-solutions/jobbergate?label=issues&logo=github&style=plastic"
+       alt="github issues"
```
Comment (dusktreader) [inline jobbergate-cli/README.rst:10]: issues and PRs links should stay on the top-level.

**Review by dusktreader — CHANGES_REQUESTED** [review]

File: jobbergate-api/README.rst (hunk 1 above)
Comment (fschuch) [reply jobbergate-api/README.rst]: solved in [2475c7c](https://github.com/omnivector-solutions/jobbergate/pull/155/commits/2475c7cb2f61a1e952c2e8ef24e32f0f2eb6dc1a)

File: jobbergate-cli/README.rst (hunk 2 above)
Comment (fschuch) [reply jobbergate-cli/README.rst]: solved in [2475c7c](https://github.com/omnivector-solutions/jobbergate/pull/155/commits/2475c7cb2f61a1e952c2e8ef24e32f0f2eb6dc1a)

**Review by dusktreader — APPROVED** [review]

**Comment by dusktreader** [comment]
Task linked: [ARMADA-712 Enhance Jobbergate's appearance on PyPI](https://app.clickup.com/t/18022949/ARMADA-712)

**Pull request merged**